### PR TITLE
[#168189131] Add info about communication with PagoPA and kubernetes …

### PR DESCRIPTION
**Pull request by LucaPrete**
…deployments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,16 @@ PAGOPA_PASSWORD=nopassword yarn start
 
 Also be sure to use a valid configuration for REDIS server or install it on local machine.
 
+## Real-world deployments and Kubernetes settings
```
Comment (cloudify) [inline README.md]: you mean production deployments?

File: README.md [hunk 2]
```diff
@@ -78,6 +78,16 @@ PAGOPA_PASSWORD=nopassword yarn start
...
+
+In real-world deployments, the application communicates with PagoPA performing a TLS mutual authentication and establishing a TLS encrypted tunnel.
+
+Authentication and tunnel mechanisms are transparent for the pagopa-proxy application and are offloaded to third-party components.
+
+Normally, the pagopa-proxy application is deployed as a set of Kubernetes resources. A specific *helm-chart* has been developed and currently used to deploy the application. The chart includes some configuration files that deploy a sidecar container that manages the authentication and encryption mechanisms with PagoPA.
```
Comment (cloudify) [inline README.md]:
```suggestion
Normally, the `pagopa-proxy` application is deployed as a set of Kubernetes resources. A specific *helm-chart* has been developed and currently used to deploy the application. The chart includes some configuration files that deploy a sidecar container that manages the authentication and encryption mechanisms with PagoPA.
```

File: README.md [hunk 3]
```diff
@@ -78,6 +78,16 @@ PAGOPA_PASSWORD=nopassword yarn start
...
+
+More information about how the pagopa-proxy application communicates with PagoPA, and how to deploy the helm-chart can be found in the [pagopa-proxy helm-chart folder in the io-infrastructure-post-config repository](https://github.com/teamdigitale/io-infrastructure-post-config/tree/master/pagopa-proxy).
```
Comment (cloudify) [inline README.md]:
```suggestion
More information about how the `pagopa-proxy` application communicates with PagoPA, and how to deploy the helm-chart can be found in the [pagopa-proxy helm-chart folder in the io-infrastructure-post-config repository](https://github.com/teamdigitale/io-infrastructure-post-config/tree/master/pagopa-proxy).
```

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#168189131](https://www.pivotaltracker.com/story/show/168189131): Add info in pagopa-proxy readme about the connection to pagopa and helm-chart deployments (point to the readme in io-infrastructure-post-config/pagopa-proxy)

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d686ffca49d65faeb7e31549d3efb1fb7be8c72e
</p>

File: README.md [hunk 4]
```diff
@@ -78,6 +78,16 @@ PAGOPA_PASSWORD=nopassword yarn start
 
 Also be sure to use a valid configuration for REDIS server or install it on local machine.
 
+## Production deployments and Kubernetes settings
+
+In real-world deployments, the application communicates with PagoPA performing a TLS mutual authentication and establishing a TLS encrypted tunnel.
```
Comment (cloudify) [inline README.md]:
```suggestion
In production deployments, the application communicates with PagoPA performing a TLS mutual authentication and establishing a TLS encrypted tunnel.
```

**Pull request merged**